c/CMakeLists.txt

```diff
@@ -1,56 +1,19 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.2)
 project(libsbp)
 
-# Setup flags for Code Coverage build mode
-set(CMAKE_CXX_FLAGS_COVERAGE "${CMAKE_CXX_FLAGS_DEBUG} --coverage" CACHE STRING
-    "Flags used by the C++ compiler for building with code coverage."
-    FORCE )
-set(CMAKE_C_FLAGS_COVERAGE "${CMAKE_C_FLAGS_DEBUG} --coverage" CACHE STRING
-    "Flags used by the C compiler for building with code coverage."
-    FORCE )
-SET(CMAKE_EXE_LINKER_FLAGS_COVERAGE
-    "${CMAKE_EXE_LINKER_FLAGS_DEBUG} --coverage" CACHE STRING
-    "Flags used for linking binaries with code coverage."
-    FORCE )
-set(CMAKE_SHARED_LINKER_FLAGS_COVERAGE
-    "${CMAKE_SHARED_LINKER_FLAGS_DEBUG} --coverage" CACHE STRING
-    "Flags used by the shared libraries linker during builds with code coverage."
-    FORCE )
-mark_as_advanced(
-    CMAKE_CXX_FLAGS_COVERAGE
-    CMAKE_C_FLAGS_COVERAGE
-    CMAKE_EXE_LINKER_FLAGS_COVERAGE
-    CMAKE_SHARED_LINKER_FLAGS_COVERAGE )
-# Update the documentation string of CMAKE_BUILD_TYPE for GUIs
-set(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}" CACHE STRING
-  "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel Coverage."
-   FORCE )
-
-# Set project version using Git tag and hash.
-execute_process(
-  COMMAND git describe --dirty --tags --always
```

Comment:
so we've totally abandoned automated versioning?

Reply:
If this was being used, it can be put back in. But I couldn't see a way the `VERSION` variable could be used. It's not set to the parent scope, and it's not being used to set the `VERSION` property. It appeared to be only used to print out the version string while `cmake` was running.

Reply:
Cool - I am all for terminated unused and unneeded code with extreme prejudice.  Just want to make sure we don't screw up someone's release process or something, since libsbp is submoduled in a lot of places.